vision/api/VisionTest/ProductSearchTest.cs

```diff
@@ -1,318 +1,365 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 using Xunit;
 
 namespace GoogleCloudSamples
 {
-    public class ProductSearchTest : IDisposable
+    public class ProductSearchTestsBase
     {
-        private const string REGION_NAME = "us-west1";
-        private const string PRODUCT_ID = "fake_product_id_for_testing_1";
-        private const string PRODUCT_DISPLAY_NAME = "fake_product_display_name_for_testing";
-        private const string PRODUCT_CATEGORY = "apparel";
-        private const string PRODUCT_ID_2 = "fake_product_id_for_testing_2";
-        private const string PRODUCT_SET_ID = "fake_product_set_id_for_testing";
-        private const string PRODUCT_SET_DISPLAY_NAME = "fake_product_set_display_name_for_testing";
-        private const string REF_IMAGE_ID = "fake_ref_image_id";
-        private const string REF_IMAGE_GCS_URI = "gs://cloud-samples-data/vision/product_search/shoes_1.jpg";
-        private const string CSV_GCS_URI = "gs://cloud-samples-data/vision/product_search/product_sets.csv";
-        private const string IMAGE_URI_1 = "shoes_1.jpg";
-        private const string IMAGE_URI_2 = "shoes_2.jpg";
-        private const string SEARCH_FILTER = "style=womens";
+        public string REGION_NAME { get; private set; } = "us-west1";
```

Comment:
Any reason these are in SHOUTY_CASE? Would prefer RegionName etc.

It feels like not all of these need to be instance properties, and not all need to be settable. Would be good to clarify that.

Reply:
I converted from SHOUTY_CASE to TitleCase.

Figuring out the exact minimal visibility required for each variable would be ideal, but just doesn't seem like a valuable use of my time.